src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,15 +209,43 @@ function toggleCollapse(e, pans) {
     var expand = 'Expand';
     var collapse = 'Collapse';
     var panels = pans || $('div.panel-collapse');
+    var shouldExpandPanels = $(e).html().trim().startsWith(expand);
```

Comment:
Yup. Here it is checking the text whether to expand the panels. However, the text here does not indicate the status or panels(though by inference we know the panels are supposed to be collapsed). Also, for expand panels button(the one that expands all panels), the button can be clicked regardless of the status of the panels.(Panels can all be collapsed and clicking on the button has no effect other than chaning the status of the button)  Therefore, I think shouldExpandPanels is a better reflection of the purpuse of the $(e).html().trim().startsWith(expand);